ModulBot/Controllers/MessageController.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Threading.Tasks;
+using Telegram.Bot.Types;
+using Microsoft.AspNetCore.Mvc;
+using Telegram.Bot.Types.Enums;
+using ModulBot.Models;
+using System.Linq;
+using ModulBot.Commands;
+using ModulBot.Cards;
+namespace ModulBot.Controllers
+{
+    [Route("api/message/update")]
+    public class MessageController : Controller
+    {
+        [HttpPost]
+        public async Task<OkResult> Post([FromBody]Update update)
+        {
+            if (update == null)
+            {
+                return Ok();
+            }
+            else
+            {
+                await MessageAction(update);
+                return Ok();
+            }
+        }
+
+        public static async Task MessageAction(Update update)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor this method to reduce its Cognitive Complexity from 29 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS3776)